### PR TITLE
Deprecate styled components

### PR DESCRIPTION
**Pull request by brianfleming**
## Description

We only had a few references left to styled-components. This PR removes the last few, transitions them to css files and removes the styled-component packages.

## Type of change

- [x] Performance improvement and/or refactoring (non-breaking change that keeps existing functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Security mitigation or enhancement
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Automated test (add or update automated tests)

## How has this been tested?

Updated all the necessary test and manually verified all the changes visually

## Things to pay attention to during code review

Not much, things should look exactly the same

## Checklist

- [x] I have performed a self-review of my own code
- [x] I've made sure my branch is runnable and given good testing steps in the PR description
- [x] I considered secure coding practices when writing this code. Any security concerns are noted above.
- [x] I have commented my code in hard-to-understand areas, if any
- [ ] I have made needed changes to the README
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If I implemented any new components, they are self-contained, their `propTypes` are declared and they use React Hooks and, if data-fetching is required, they use Relay Modern with fragment containers
- [ ] If my components involve user interaction - specifically button, text fields, or other inputs - I have added a [BEM-like class name](https://meedan.atlassian.net/wiki/spaces/ENG/pages/1327628289/Naming+conventions+for+interactive+elements) to the element that is interacted with
- [x] To the best of my knowledge, any new styles are applied according to the design system
- [ ] If I added a new external dependency, I included a rationale for doing so and an estimate of the change in bundle size (e.g., checked in https://bundlephobia.com/)

### CODE REVIEW

File: src/app/components/media/MediaCardLarge.js [hunk 1]
```diff
@@ -55,85 +46,86 @@ const MediaCardLarge = ({
...
-          </Box>
-        ) : null }
-        { type === 'UploadedImage' ? (
-          <ImageMediaCard
-            projectMedia={projectMedia}
-            imagePath={media.embed_path}
-            currentUserRole={currentUserRole}
-            superAdminMask={superAdminMask}
+    <div
+      inModal={inModal}
```
Comment (amoedoamorim) [inline src/app/components/media/MediaCardLarge.js:50]: inModal isn't helpful here anymore

Comment (brianfleming) [reply src/app/components/media/MediaCardLarge.js]: 🙃 uhhh..yeah true. removed!

File: src/app/components/source/SourcePicture.js [hunk 2]
```diff
@@ -154,13 +115,22 @@ class SourcePicture extends Component {
...
+      <div
         alt="avatar"
         size={this.props.size}
         type={this.props.type}
-        className={this.props.className}
+        className={cx(
+          styles['source-picture'],
+          {
+            [this.props.className]: true,
+            [styles.sizeLarge]: this.props.size === 'large',
```
Comment (amoedoamorim) [inline src/app/components/source/SourcePicture.js:126]: This is more of a question than a review comment, but how do you approach style class naming? I'm not always sure when to name my class using camelCase, PascalCase or kebab-case

Comment (brianfleming) [reply src/app/components/source/SourcePicture.js:126]: yeah, honestly its something we/I am not consistent about, but, but maybe as a front-end team we can come to some sort of standard.
Random not complete thoughts that don't lead to any conclusion, but 🤷 :
- sometimes when doing refactoring I maintain "whatever was there before" to keep the PR clearer
- camelCase is more like our js, so sometimes it more closely matches the other parts of the code
- kebab-case is more of a css standard approach that most people prefer/linters

**Review by amoedoamorim — APPROVED** [review]

**Pull request merged**